### PR TITLE
add sample podutil config, ditch bootstrap config

### PR DESCRIPTION
**Pull request by krzyzacy**
So I can have some reference and new repo should adopt podutils.

/area documentation
/assign @spiffxp @cjwagner @BenTheElder

### CODE REVIEW

**Comment by krzyzacy** [comment]
`added kind/documentation  and removed approved  size/M  labels 2 minutes ago`
/shrug

File: README.md [hunk 1]
```diff
@@ -108,24 +108,22 @@ you'll need to do the following:
...
-            - --
-            - make
-            - test
+    - name: foo-repo-presubmit-test
+      agent: kubernetes
+      decorate: true
+      spec:
+        containers:
+        - image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
+          command:
```
[the remark this answers is not in the document]
Comment (krzyzacy) [reply README.md]: done

File: README.md [hunk 2]
```diff
@@ -108,24 +108,24 @@ you'll need to do the following:
...
-            - --timeout=90
-            - --scenario=execute
-            - --
-            - make
-            - test
+    - name: foo-repo-presubmit-test
+      decorate: true
+      spec:
+        containers:
+        - image: where/your/image/is
```
Comment (BenTheElder) [inline README.md]: maybe an example image or something? this is not a very typical looking docker image name..

Comment (krzyzacy) [reply README.md]: sure, promoted kubekins

**Pull request merged**